tests/algorithms/test_base.py

```diff
@@ -41,7 +41,10 @@ def test_check_fixed_horizon_flag(custom_logger):
 
 
 def _make_and_iterate_loader(*args, **kwargs):
-    loader = base.make_data_loader(*args, **kwargs)
+    # our pytype version doesn't understand optional arguments in TypedDict
```

Comment:
Probably fine to upgrade pytype -- we had the habit of specifying test-only dependencies as `==` so that CI wouldn't spuriously break (whereas for install dependencies we *want* it to break if there's some upstream incompatibility, since it'll hurt our users)